packages/cozy-realtime/README.md

```diff
@@ -44,6 +44,37 @@ In order to listen to online and offline events, you should install the [Network
 
 ## Example
 
+### With CozyClient
+
+CozyRealtime comes with a plugin for CozyClient. This is the recommended way to use it: 
+```js
+//instanciation
+import { RealtimePlugin } from 'cozy-realtime'
+import { CozyClient} from 'cozy-client'
+
+const client = new CozyClient({})
+
+client.registerPlugin(RealtimePlugin)
+
+//usage
+
+this.realtime = client.plugins.realtime
```

Comment:
The this here is undefined. Maybe it would make more sense to have the example in the context of a React component, with componentDidMount / componentWillUnmount ?